### ISSUE TITLE
Do we need an awards topic

### ISSUE THREAD

**Issue by markcmiller86** [betterscientificsoftware/bssw.io#2016]
I am creating an even for [US-RSE Community Awards](https://us-rse.org/community-awards/). We have Gordon Bell awards, R&D-100 awards. There are others.

If we created an `awards` topic, I would linclude it under `collaboration` I think.

**Comment by rinkug** [comment]
I dont think we need awards topic. These are all events mostly and most articles are one-off.
